goals.md

```diff
@@ -9,29 +9,29 @@ Based on this, we would like to bring the goals to the TODO Group for review and
 
 The goals will be revisited and updated yearly.
 
-## 2022
+## [PROPOSED] 2022
 
-### 1. Cultivate growth and adoption of OSPOs
+### 1 Cultivate growth and adoption of OSPOs
 
-* Release at least 6 new OSPO case studies
-* Launch 6 new OSPO guides
-* Host 12 OSPOlogy sessions
-* Build out and run OSPO Survey 2022
-* Create and publish a OSPO Maturity Model
+- Release at least 6 new OSPO case studies
+- Launch 6 new OSPO guides
```

Comment:
We are having more folks interested in OSPOs this year + we are working on a contribution guideline and a process to make more clear and easier these contributions 👍 I like the idea to have an ambitious goal and take corrective actions by the end of the year if necessary, rather than reduce the number without trying it first 🙂

Reply:
Goal suggestion: Make it easier to produce high quality TODO group guides and case studies